cache/s3proxy/s3proxy.go

```diff
@@ -186,7 +186,6 @@ func (c *s3Cache) Get(kind cache.EntryKind, hash string) (io.ReadCloser, int64,
 
 func (c *s3Cache) Contains(kind cache.EntryKind, hash string) (bool, int64) {
 	size := int64(-1)
-
```

Comment:
nitpick: please remove this stray change

Reply:
Done